src/interfaces/swig/ModelSelection.i

```diff
@@ -10,6 +10,8 @@
 %newobject CParameterCombination::leaf_sets_multiplication();
 %newobject CModelSelectionParameters::get_combinations();
 %newobject CModelSelectionParameters::get_single_combination();
+%newobject ParameterNode::attach();
```

Comment:
I get that we need attach, but why would we need to call `next_combination` from swig?

Reply:
that's just for debugging right now